pkg/operator/ceph/operator.go

```diff
@@ -48,15 +48,16 @@ const (
 	provisionerNameLegacy = "rook.io/block"
 )
 
-var logger = capnslog.NewPackageLogger("github.com/rook/rook", "operator")
+var (
+	LogLevel capnslog.LogLevel
```

Comment:
nit: the var isn't needed outside this package so it can be lower case
```suggestion
	logLevel capnslog.LogLevel
```

Reply:
done